### PR TITLE
Update scheduling example and single-source solution steps

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
Redo the scheduling example to build BQMs with penalties without using the dwavebinarycsp package. Also substituted an updated single instance of the solutions steps for the duplications in all examples.

### CODE REVIEW

File: docs/examples/map_dqm.rst [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
-demonstrate `Leap <https://cloud.dwavesys.com/leap/>`_\ 's hybrid discrete 
-quadratic model (:term:`DQM`) solver, which enables you to solve problems 
+This example solves the same map coloring problem of :ref:`map_kerberos` to
+demonstrate `Leap <https://cloud.dwavesys.com/leap/>`_\ 's hybrid discrete
+quadratic model (:term:`DQM`) solver, which enables you to solve problems
 of arbitrary structure and size for variables with **discrete** values.
 
-See :ref:`map_kerberos` for an description of the map coloring 
-:doc:`constraint satisfaction problem </concepts/csp>` (CSP). 
+See :ref:`map_kerberos` for an description of the map coloring
```
Comment (hhtong) [inline docs/examples/map_dqm.rst:12]: a description

File: docs/examples/map_dqm.rst [hunk 2]
```diff
@@ -85,16 +85,16 @@ Graph G now represents states as vertices and each state's neighbors as shared e
 >>> borders = G.edges       # doctest: +SKIP
 
 You can now create a :class:`dimod.DiscreteQuadraticModel` class to represent
```
Comment (hhtong) [inline docs/examples/map_dqm.rst:87]: create a :class:`dimod.DiscreteQuadraticModel` object?

File: docs/examples/scheduling.rst [hunk 3]
```diff
@@ -86,42 +122,118 @@ find solutions that meet all the constraints.
...
-
-   def scheduling(time, location, length, mandatory):
-       if time:                                 # Business hours
-           return (location and mandatory)      # In office and mandatory participation
-       else:                                    # Outside business hours
-           return ((not location) and length)   # Teleconference for a short duration
+Represent Constraints as Penalties
+----------------------------------
+
+You can represent constraints as a BQMs using :ref:`penalty_sdk` in many
```
Comment (hhtong) [inline docs/examples/scheduling.rst:128]: as BQMs

**Review by hhtong — APPROVED** [review]
LGTM

**Review by randomir — APPROVED** [review]
LGTM.

**Pull request merged**